storage/google/cloud/storage/bucket.py

```diff
@@ -1377,6 +1435,38 @@ def location(self, value):
         warnings.warn(_LOCATION_SETTER_MESSAGE, DeprecationWarning, stacklevel=2)
         self._location = value
 
+    @property
+    def location_type(self):
+        """Retrieve or set the location type for the bucket.
+
+        See https://cloud.google.com/storage/docs/storage-classes
+
+        :setter: Set the location type for this bucket.
+        :getter: Gets the the location type for this bucket.
+
+        :rtype: str or ``NoneType``
+        :returns:
+            If set, one of :attr:`MULTI_REGION_LOCATION_TYPE`,
+            :attr:`REGION_LOCATION_TYPE`, or :attr:`DUAL_REGION_LOCATION_TYPE`,
+            else ``None``.
+        """
+        return self._properties.get("locationType")
+
+    @location_type.setter
```

Comment:
Location type is actually a read-only field and updates to it are ignored by the back-end, should it have a setter here?

Reply:
The setter provides the only way to create a new bucket with a non-default `location_type`, e.g.:

```python
from google.cloud.storage.bucket import Bucket

bucket = client.bucket("<BUCKET_NAME_HERE>")
bucket.location_type = Bucket.DUAL_REGION_LOCATION_TYPE
bucket.create()
```

Reply:
Do we need to add `location_type` as a parameter for `Bucket.create`, and just remove the `location_type` setter?  For the `location` property, we left the setter but emit a deprecation warning, suggesting to use the `location` parameter to `Bucket.create`, but that was for backward-compatibility.

Reply:
The field is actually ignored by the backend even on create

Reply:
`location_type` setter dropped in 8be3cb1.